lib/src/main/java/uk/gov/di/ipv/cri/address/library/helpers/EventProbe.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.di.ipv.cri.address.library.helpers;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.cloudwatchlogs.emf.logger.MetricsLogger;
+import software.amazon.cloudwatchlogs.emf.model.DimensionSet;
+import software.amazon.lambda.powertools.metrics.MetricsUtils;
+
+import java.util.Map;
+
+public class EventProbe {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+    private static final MetricsLogger metricsLogger = MetricsUtils.metricsLogger();
+
+    public EventProbe log(Level level, Exception e) {
```

Comment:
Is it worth having "helper" methods such as "debug", "info", "warning" etc as per log4j?

Reply:
I'm ok with getting this extra methods added when we need them, but for the SessionHandler the method above is all that's needed.